### PR TITLE
Additional styles for extended content table customization

### PR DESCRIPTION
**Pull request by broadwell**
These are the extra CSS styles needed to enable the table row and cell customizations requested in Jarek's email and mentioned in the comment to #499 -- namely, being able to specify the background color of a row or cell (choosing from our palette of beige or white) and to specify the font of a cell (regular serif or else the all-caps green sans font). All of the other requested customization options can be handled via attributes attached directly to the HTML, as explained on the wiki.

### CODE REVIEW

**Comment by simonwiles** [comment]
These extra styles look great, but I'm concerned that the class names are not great.  Technically, CSS class names, like all other CSS identifies, [are not allowed to begin with two dashes](https://www.w3.org/TR/CSS21/syndata.html#value-def-identifier) (although it seems the major browsers, at least, will let you get away with it).  The hyphens also introduce confusion because they're reserved for use in [css-variables](https://drafts.csswg.org/css-variables/).

I suspect you were seeking to emulate this kind of thing?

    <tr class="content-table__row[--header]>

In our codebase this is spelled out long-hand, but this kind of construct can be written in SCSS like this:

    .content-table__row {
      ...
      &--beige {    
    ...

Note, no `.` – the `--beige` is simply appended to the class name to create a new class.  This is commonly seen in [BEM-style CSS methodology](http://getbem.com/introduction/) (which I'm not a big proponent of, or anything, but it has its strengths).

The problem with this approach, however, is that it doesn't scale well -- if you have more than one optional modifier like this, they become cumulative -- they cannot be skipped and they have to be specified in the correct order to work.

**TL;DR:**  The hyphens at the beginning of the class names aren't adding any value or doing any work in respect of isolation or encapsulation, and are technically invalid and may cause unexpected errors in unexpected places.  Can we just drop them?

**Comment by broadwell** [comment]
All good points! You're right, I was trying to get BEM-style chaining without all the extra typing, but just removing the hyphens makes a whole lot more sense. Will update and rebase.

**Comment by broadwell** [comment]
This should be ready for prime time now.
